.github/workflows/ipc_fuzzer.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+# This is NOT how to properly compile the fuzzer, this is only to detect
+# earlier and faster problems like
+# https://github.com/thesofproject/sof/pull/3824 (288e228) and
+# https://github.com/thesofproject/sof/pull/4118 (758e7be)
+#
+# In other words, when reverting the oss-fuzz parts of these commits
+# this build fails as desired.
+#
+# For the actual fuzzer see tools/oss-fuzz/README.
+
+name: IPC fuzzer compile test
+
+# 'workflow_dispatch' allows running this workflow manually from the
+# 'Actions' tab
+# yamllint disable-line rule:truthy
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  ipc-fuzzer-build:
+    runs-on: ubuntu-latest
+    steps:
+      # See https://google.github.io/oss-fuzz/getting-started/continuous-integration/
```

Comment:
> line too long (85 > 80 characters)

:-(

This is a problem because the yamllint action is not incremental/"smart" so it would print this warning on every single pull request after this get merged.

`# yamllint disable-line rule:line-length` should fix it.

Alternatively, in this case you could just shift if left and it should (barely) fit.